sky/cli.py

```diff
@@ -1706,13 +1722,51 @@ def status(all: bool, refresh: bool, show_spot_jobs: bool, clusters: List[str]):
                           show_all=False,
                           limit_num_jobs_to_show=not all,
                           is_called_by_user=False))
-        click.echo(f'{colorama.Fore.CYAN}{colorama.Style.BRIGHT}Clusters'
-                   f'{colorama.Style.RESET_ALL}')
+        if ip:
+            if len(clusters) != 1:
+                with ux_utils.print_exception_no_traceback():
+                    plural = 's' if len(clusters) > 1 else ''
+                    raise RuntimeError(
```

Comment:
nit - This and L1746 might be better suited to be a `ValueError`, since the input is incorrect (there's no cluster specified/the cluster specified does not exist)